src/main/java/br/application/inviteu/SecurityConfiguration.java

```diff
@@ -17,12 +22,28 @@ public static BCryptPasswordEncoder passwordEncoder(){
       return new BCryptPasswordEncoder();
    }
 
+   @Autowired
+   private UserLoginService userLoginService;
+
+   @Autowired
+   private UserRepository userRepository;
+
+   @Override
+   public UserDetailsService userDetailsServiceBean() throws Exception{
+      return new UserLoginService(userRepository);
+   }
+
    @Override
    protected void configure(HttpSecurity http) throws Exception{
       http.authorizeRequests()
-               .antMatchers("/", "/h2-console/**").permitAll()
-               .and()
-               .httpBasic();
+            .antMatchers("/", "/login", "/h2-console/**").permitAll()
+            //.antMatchers("/api/roles/**").hasAnyAuthority("ADMIN")
```

Comment:
Esses comentários serão utilizados posteriormente, quando for para valer as permissões que utilizaremos?

Reply:
Isso mesmo, eu deixei comentado pra não ter que ficar se preocupando com autenticação enquanto desenvolvemos o projeto.